docs/new-feature-tutorial.md

```diff
@@ -27,14 +59,16 @@ detailed explanation of how everything works.
 
 **Test your changes:** Once you've run the migration, restart memcached
 on your development server (`/etc/init.d/memcached restart`) and then
-restart `run-dev.py` to avoid interacting with cached objects.
+restart the development server (`./tools/run-dev.py`) to avoid interacting
+with cached objects.
 
 ### Backend changes
 
-For most new features/settings, the `property_types` framework handles
-most of this automatically, but it's valuable to understand the flow
-even if the `property_types` framework means you don't have to write
-any code.
+For most new features/settings, there is a framework, referred to as
```

Comment:
Maybe better would be "We have framework that automatically handles many of the these steps for the most common types of settings..."